webapp/test_run_handler.go

```diff
@@ -31,6 +31,13 @@ func handleTestRunGet(w http.ResponseWriter, r *http.Request) {
 	}
 	sourceURL := fmt.Sprintf(`/api/runs?max-count=%d`, maxCount)
 
+	labels := ParseLabelsParam(r)
+	if labels != nil {
+		for label := range labels.Iterator().C {
+			sourceURL = sourceURL + "&label=" + label.(string)
```

Comment:
Is `.(string)` a type assertion? Could it fail?

Reply:
Yes it's an assertion; no it can't fail unless the set items aren't strings (which, they are).